readme.rst

```diff
@@ -258,6 +258,26 @@ Add redirects
 You can add redirects to make sure existing links and bookmarks continue working when you move files around.
 To do so, specify the old and new paths in the ``redirects`` setting of the ``custom_conf.py`` file.
 
+Configure included extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The starter pack includes a set of extensions that we deem useful for all documentation sets.
+They are pre-configured as needed, but you can customise their configuration in the  ``custom_conf.py`` file.
+
+The following extensions are included automatically:
```

Comment:
s/automatically/by default/ ?

Reply:
I don't think that works, because they cannot be removed. But I updated it to "are always included".